app/controllers/repositories_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
     if @repository.private? && !@check_passed
       redirect_to password_entry_repository_path(@repository.user_username, @repository.slug) and return
     end
-    @photos = @repository.photos.first(5)
+    @photos = @repository.photos.first(5) rescue nil
```

Comment:
I would prefer to not rescue nil here and instead do `@photos = @repository.photos&.first(5)`, possibly even `@photos = @repository.photos&.first(5) || []` so that you can still do `@photos.each` without hitting a nil error

Reply:
done